bots/dev/pr_comment_parser.py

```
@@ -0,0 +1,213 @@
+def get_pr_comments(pr_number: int, repo: str) -> List[Dict]:
+    """Fetch all comments from a GitHub PR using gh CLI.
+
+    Parameters:
+        pr_number (int): The PR number
+        repo (str): Repository in format "owner/repo"
+
+    Returns:
+        List[Dict]: List of comment dictionaries
+    """
+    cmd = ["gh", "pr", "view", str(pr_number), "--repo", repo, "--json", "comments"]
+    result = subprocess.run(cmd, capture_output=True, text=True, encoding="utf-8", check=True)
```

Ruff (0.13.3)

`subprocess` call: check for execution of untrusted input

(S603)